Tasks/DotNetCoreCLIV2/pushcommand.ts

```diff
@@ -99,8 +99,16 @@ export async function run(): Promise<void> {
                 tempNuGetPath,
                 false /* useNugetToModifyConfigFile */);
 
-            const internalFeedId = tl.getInput('feedPublish');
-            feedUri = await nutil.getNuGetFeedRegistryUrl(packagingLocation.DefaultPackagingUri, internalFeedId, null, accessToken, /* useSession */ true);
+            const feedProject = tl.getInput('feedPublish');
+            var project = null;
+            var internalFeedId = feedProject;
+            if(feedProject && feedProject.includes("/")) {
```

Comment:
Should this be a common method across two dot net core cli task commands

Reply:
Fixed